tasks/tests/test_delete_task.py

```diff
@@ -0,0 +1,33 @@
+from tasks.models import Task
+import pytest
+
+
+@pytest.mark.django_db
+class TestDeleteTask:
+    def test_no_authenticated_user_delete_task(self, client, task_1):
+        response = client.get(f'/tasks/delete/{task_1.id}')
+        assert response.status_code == 302
+        assert response.url == '/'
+        Task.objects.get(id=task_1.id)
+
+    def test_account_delete_task(self, client, task_1, employee_1):
+        client.force_login(employee_1.user)
+        response = client.get(f'/tasks/delete/{task_1.id}')
+        assert response.status_code == 302
+        assert response.url == '/'
+        Task.objects.get(id=task_1.id)
+
+    def test_manager_different_team_delete_task(self, client, task_3, manager_1):
+        client.force_login(manager_1.user)
+        response = client.get(f'/tasks/delete/{task_3.id}')
+        assert response.status_code == 302
+        assert response.url == '/'
+        Task.objects.get(id=task_3.id)
+
+    def test_manager_delete_task(self, client, manager_1, task_1):
+        client.force_login(manager_1.user)
+        response = client.get(f'/tasks/delete/{task_1.id}')
+        assert response.status_code == 302
+        assert response.url == '/tasks/'
+        with pytest.raises(Exception):
+            Task.objects.get(id=task_1.id)
```

Comment:
Can you also add a verification here that the related comments are being deleted too please?

Reply:
i dont think they do ...

Reply:
I'm pretty sure it should be deleted.. when you try to delete a task that has comments through the admin section, it will ask you if you are sure about it because the related comments will be deleted. Shouldn't it have the same behavior here?

Reply:
![image](https://user-images.githubusercontent.com/66386688/147644417-d5b185c2-8d39-42a3-aa24-55821e8c26b3.png)

Reply:
I deleted the comments and updated the test...